### PR TITLE
replace tables with datatables #225

### PR DESCRIPTION
**Pull request by samaradel**
- #225

### CODE REVIEW

File: client/src/views/K8s.vue [hunk 1]
```diff
@@ -291,21 +231,65 @@ export default {
...
+      {
+        title: "RAM (GB)",
+        key: "mru",
+      },
+      {
+        title: "CPU",
+        key: "cru",
+      },
+      {
+        title: "Yggdrasil IP",
```
Comment (rawdaGastan) [inline client/src/views/K8s.vue:262]: I think we don't have a reference for ygg anymore?

**Review by rawdaGastan — COMMENTED** [review]
can we add a color to the headers? 👀

File: client/src/views/K8s.vue [hunk 2]
```diff
@@ -156,10 +156,11 @@
                       icon="fa-solid fa-eye"
                     />
                   </template>
-                  <v-card width="50%" class="mx-auto pa-5">
-                    <v-icon class="ml-auto" @click="dialog = false"
-                      >mdi-close</v-icon
-                    >
+                  <v-card
```
Comment (rawdaGastan) [inline client/src/views/K8s.vue:159]: why close button removed?

Comment (samaradel) [reply client/src/views/K8s.vue:159]: we have many dialogs for workers it fires all together it closes when you click outside, will figure it out later https://github.com/codescalers/cloud4students/issues/238

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**